packages/usa-tooltip/src/index.js

```diff
@@ -171,7 +171,7 @@ const showToolTip = (tooltipBody, tooltipTrigger, position) => {
     e.style.left = `${
       tooltipTrigger.offsetLeft + tooltipTrigger.offsetWidth + TRIANGLE_SIZE
     }px`;
-    e.style.margin = `-${topMargin / 2}px 0 0 0`;
+    e.style.margin = `${topMargin / -2}px 0 0 0`;
```

Comment:
## OPTIONAL - Negative denominator

Should we move the negative number back to the numerator for consistency with the other calculations?

```suggestion
    e.style.margin = `${-topMargin / 2}px 0 0 0`;
```

Reply:
@mahoneycm if it's inconsistent I wouldn't consider the request optional. Unless I'm misunderstanding the before/after.

Reply:
Updated my request to remove "optional"

Reply:
> We should move the negative number back to the numerator for consistency with the other calculations.

I updated this in 2596c38.